packages/trader/src/App/Components/Form/fieldset.jsx

```diff
@@ -3,7 +3,16 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { Popover } from '@deriv/components';
 
-const Fieldset = ({ children, className, header, header_tooltip, is_center, onMouseEnter, onMouseLeave }) => {
+const Fieldset = ({
+    children,
+    className,
+    header,
+    header_tooltip,
+    is_center,
+    onMouseEnter,
+    onMouseLeave,
+    is_tooltip_disabled,
```

Comment:
nitpick but seems like the list is in alphabetical order 😄

Reply:
ah yes 👍